client/src/components/WaitingRoom/index.js

```diff
@@ -76,7 +100,19 @@ class WaitingRoomBase extends Component {
         </nav>
         <button disabled= {isInvalid} type="button" onClick={() =>
```

Comment:
looking @ line 86.. does this mean the host cannot press this button?

Reply:
no the host would still be able to start because they're inside the game.

Reply:
wait
(isHost && started === false && joined === true) would be true then

Reply:
the functionality was working, but thanks for pointing it out because it did need to be fixed